src/components/views/settings/devices/FilteredDeviceList.tsx

```diff
@@ -185,9 +200,15 @@ export const FilteredDeviceList =
         saveDeviceName,
         onSignOutDevices,
         onRequestDeviceVerification,
+        setPusherEnabled,
+        supportsMSC3881,
     }: Props, ref: ForwardedRef<HTMLDivElement>) => {
         const sortedDevices = getFilteredSortedDevices(devices, filter);
 
+        function getPusherForDevice(device: DeviceWithVerification): IPusher | null {
```

Comment:
Type is wrong here, `find` will return undefined not null when no pusher is found